tests/datasets-state-isnotset/test.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  min-version: 5.0.0
+
+pcap: ../datasets-05-state/input.pcap
+
+args:
+  - --data-dir ./output
```

Comment:
This is not portable
```
[15991] 5/10/2019 -- 08:24:13 - (suricata.c:1876) <Error> (ParseCommandLine) -- [ERRCODE: SC_ERR_LOGDIR_CMDLINE(117)] - The data directory "./output" supplied at the commandline (-d ./output) doesn't exist. Shutting down the engine.
```

Reply:
Ah, I see.  Maybe we should consider supporting environment vars in the test.yaml, should simplify this case.